composer.json

```
@@ -10,27 +10,35 @@
       "homepage": "https://thephp.foundation"
     }
   ],
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/pronskiy/dflydev-canal"
+    },
+    {
+      "type": "vcs",
+      "url": "https://github.com/sculpin/sculpin"
+    }
+  ],
   "require": {
-    "php": "^8.0",
-    "sculpin/sculpin": "^3.1",
-    "twig/twig": "2.15.3",
+    "php": "^8.2",
     "ext-gd": "*",
-    "ext-dom": "*"
+    "ext-dom": "*",
+    "twig/twig": "2.15.3",
+    "sculpin/sculpin": "dev-main#467ffd4d7d0e3b28c8b8234d7f29bb3c8caf6c44",
```

```suggestion
    "sculpin/sculpin": "3.3.0-alpha",
```

Sculpin `3.3.0-alpha` includes sculpin/sculpin@467ffd4, and twig 3 support
(perhaps we can wait to a stable release)

---

Go ahead 👍